src/Styleguide/Components/Timer.tsx

```diff
@@ -14,10 +14,11 @@ export const Timer: React.SFC<{
   labelWithTimeRemaining?: string
   labelWithoutTimeRemaining?: string
 }> = ({ endDate, labelWithTimeRemaining, labelWithoutTimeRemaining }) => (
-  <WithCurrentTime>
-    {currentTime => {
+  <WithCurrentTime useServerAdjustment>
+    {({ currentTime, timeOffsetInMilliseconds }) => {
       const duration = moment.duration(
-        Math.max(moment(endDate).diff(moment(currentTime)), 0)
+        Math.max(moment(endDate).diff(moment(currentTime)), 0) +
+          timeOffsetInMilliseconds
```

Comment:
A child component of `WithCurrentTime` that needs to use the offset, can just use it like so.